src/scripting/CompilerTxt.js

```diff
@@ -58,8 +58,6 @@ module.exports = class CompilerTxt extends CompilerBase {
     let currentChannel = null
 
     const parseMsg = (lines) => {
-      if (!lines || lines.length === 0) return {}
```

Comment:
Shouldn't there be a security check for at least undefined ? something like
lines = lines || []
Just in case ...